examples/sample-app/README.md

```diff
@@ -0,0 +1,44 @@
+# Blockly Sample App
+
+## Purpose
+
+This app illustrates how to use Blockly together with common programming tools like node/npm, webpack, typescript, eslint, and others. You can copy and modify it as much as you'd like. It contains basic infrastructure for running, building, testing, etc. that you can use even if you don't understand how to configure the related tool yet. For creating your first application using Blockly, you can use the tools we've provided here for as long as they work for you. Eventually, you might need to customize the way your application builds, for example, and at that time you can replace the provided webpack configuration with your own.
```

Comment:
```suggestion
This app illustrates how to use Blockly together with common programming tools like node/npm, webpack, typescript, eslint, and others. The intention is that you use this to get started using your application, and then modify it over time to suite your specific needs. To facilitate this, it contains basic infrastructure for running, building, testing, etc. that you can use even if you don't understand how to configure the tools for doing that yet. 
```

The original wording felt a little clunky to me, but not sure. This one is def just a suggestion, so feel free to not accept.

Reply:
Updated the wording, lmk what you think.